docs/source/upcoming_release_notes/616-DEV:_Add_KBO_tests.rst

```diff
@@ -0,0 +1,34 @@
+618 DEV: Add KBO tests
+#################
+
+API Changes
+-----------
+- N/A
+
+Features
+--------
+- N/A
+
+Device Updates
+--------------
+- pcdsdevices/mirror.py
+  added RTD PVs to KBOMirror class for bender actuators
+
+New Devices
+-----------
+- tests/test_mirror.py
+  add prefix and lightpath tests for KBOMirror. FFMirror is an identical
+  but reduced class so did not write an additional test for that.
```

Comment:
Take a quick peek at https://raw.githubusercontent.com/pcdshub/pcdsdevices/master/docs/source/upcoming_release_notes/template-full.rst for the intention behind the `New Devices` category (and the other categories)

Reply:
I think this note belongs in `Maintenance`